src/jaxsim/api/references.py

```diff
@@ -26,7 +25,8 @@ class JaxSimModelReferences(js.common.ModelDataWithVelocityRepresentation):
     Class containing the references for a `JaxSimModel` object.
     """
 
-    input: ODEInput
+    _link_forces: jtp.Matrix
```

Comment:
Maybe we can add a comment to explain what these attributes are and which representation is used to store the link forces?

Reply:
Thanks @xela-95! Done in https://github.com/ami-iit/jaxsim/pull/307/commits/baac06cd65443c55ba2eacfc013f63b7e08d546d